OrbitGl/App.cpp

```diff
@@ -79,6 +79,7 @@ bool DoZoom = false;
 OrbitApp::OrbitApp(ApplicationOptions&& options)
     : options_(std::move(options)) {
   main_thread_executor_ = MainThreadExecutor::Create();
+  thread_pool_ = ThreadPool::Create(4 /*min_size*/, 256 /*max_size*/);
```

Comment:
Any reasoning why you particularly picked these numbers? Ok to try and experiment and change later, just want to understand if there was any rationale behind this.

Reply:
4 actions seems to be sufficient for the most things users do. the maximum is a big number so that we always spawn a thread if all others are busy, but not too big so that in the unlikely scenario when we hit it, it does not bring down the system.